### PR TITLE
Move redshift, snowflake, bigquery plugins

### PR DESCRIPTION
**Pull request by jtcohen6**
> It seems such a waste of time
> If that's what it's all about
> Mama if that's movin' up
> Then I'm movin' out
> I'm movin' out

These are private for now, while we set up automated tests and do some repo housekeeping:

- https://github.com/dbt-labs/dbt-redshift
- https://github.com/dbt-labs/dbt-snowflake
- https://github.com/dbt-labs/dbt-bigquery

### CODE REVIEW

**Comment by jtcohen6** [comment]
We ran + passed 455 integration tests on Postgres, the same number as on the `develop` branch currently.

I've commented out non-Postgres RPC integration tests. The real resolution here will come in #3755.

**Review by kwigley — APPROVED** [review]

**Comment by jtcohen6** [comment]
Planned merge: Saturday, Oct 2. We're freezing all code Friday, October 1 @ 5 pm ET. We'll incorporate any updates to adapters + integration tests in the dbt-redshift, dbt-snowflake, and dbt-bigquery repos, and then merge this PR.

File: scripts/build-dist.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ rm -rf "$DBT_PATH"/dist
 rm -rf "$DBT_PATH"/build
 mkdir -p "$DBT_PATH"/dist
 
-for SUBPATH in core plugins/postgres plugins/redshift plugins/bigquery plugins/snowflake
+for SUBPATH in core plugins/postgres
```
Comment (iknox-fa) [inline scripts/build-dist.sh:17]: Strictly speaking there's no need for a loop over a collection of one. :). NBD though

**Review by iknox-fa — APPROVED** [review]
LGTM!

**Pull request merged**